### ISSUE TITLE
react-spinners does not compile when being hoisted

### ISSUE THREAD

**Issue by sguillia** [davidhu2000/react-spinners#460]
**Describe the bug**
Demo code doesn't build when react-spinners lives in a node_modules higher up in the tree

Example tree

```
node_modules
└── react-spinners

my-app
├── src
│   ├── App.tsx
├── node_modules
└── └── react-scripts
```

This tree is a standard tree when using [yarn workspaces](https://classic.yarnpkg.com/lang/en/docs/workspaces/)
This tree respects the node module resolution algorithm

**To Reproduce**

1. cd /tmp
2. npx create-react-app my-app --template typescript
3. cd my-app
4. Copy demo code from https://www.npmjs.com/package/react-spinners into `my-app/src/App.tsx`
5. npm install react-spinners
6. mkdir ../node_modules
7. mv node_modules/react-spinners ../node_modules
8. npm run build

**Expected behavior**

Compiles

**Actual behavior**

```
Creating an optimized production build...
Failed to compile.

TS2607: JSX element class does not support attributes because it does not have a 'props' property.
    19 |       <input value={color} onChange={(input) => setColor(input.target.value)} placeholder="Color of the loader" />
    20 |
  > 21 |       <ClipLoader color={color} loading={loading} css={override} size={150} />
       |       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    22 |     </div>
    23 |   );
    24 | }

```

**Comment by davidhu2000** [comment]
@sguillia i tried to reproduce using the latest version. maybe 0.12.0 resolved this issue. can you check?

**Comment by davidhu2000** [comment]
@sguillia i'm not sure how, but this seems fixed on 0.13.0-beta.4. mind giving that a try? 

I did reproduce this issue on 0.12.0, and then tried again on 0.13 and it worked. 

i see this on 0.13
<img width="610" alt="Screen Shot 2022-06-04 at 6 20 09 PM" src="https://user-images.githubusercontent.com/15827041/172030929-58ac1e92-fb63-4f79-a4ff-a01e37d4d09b.png">

vs 0.12
<img width="660" alt="Screen Shot 2022-06-04 at 6 20 50 PM" src="https://user-images.githubusercontent.com/15827041/172030935-765276f2-43a1-47f9-b86d-7fb3ce92506c.png">

**Comment by sguillia** [comment]
@davidhu2000 It works with 0.13.0-beta.4. Thank you!
